node-api/src/decoder/bit_sequence.rs

```diff
@@ -0,0 +1,81 @@
+// This file was taken from scale-value (Parity Technologies (UK))
```

Comment:
Everything in the decoder folder was taken from [scale-value](https://crates.io/crates/scale-value) and adapted to `no_std`. No need to review in Detail, I suppose, as it's already been reviewed by parity.